kernel/kernel-api/src/main/java/io/delta/kernel/coordinatedcommits/TableDescriptor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.coordinatedcommits;
+
+import static java.util.Objects.requireNonNull;
+
+import io.delta.kernel.TableIdentifier;
+import io.delta.kernel.annotation.Evolving;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * The complete descriptor of a Coordinated Commits (CC) Delta table, including its logPath, table
+ * identifier, and table CC table configuration.
```

Comment:
```suggestion
 * The complete descriptor of a Coordinated Commits (CC) Delta table, including its logPath, table
 * identifier (if access is not path-based), and table CC table configuration.
```

Reply:
Hm... that's not _quite_ correct, right? It's not like you pass either one of the path OR the identifier, but not BOTH, to the CC. We are passing both ...

Reply:
I meant that if the access is path-based, there won't be a table identifier.

Reply:
I'll include this change in PR 3 https://github.com/delta-io/delta/pull/3798